src/Gizmos/gizmo.ts

```diff
@@ -196,7 +201,7 @@ export class Gizmo implements IDisposable {
                 effectiveNode.getWorldMatrix().decompose(undefined, this._rootMesh.rotationQuaternion!);
             }
             else {
-                this._rootMesh.rotationQuaternion!.copyFrom(this._customRotationQuaternion || new Quaternion(0, 0, 0, 1));
+                this._customRotationQuaternion ? (this._rootMesh.rotationQuaternion!.copyFrom(this._customRotationQuaternion)) : (this._rootMesh.rotationQuaternion!.set(0, 0, 0, 1));

```

Comment:
I'm not fond of using the ternary operator to hide a `if` statement when the expressions on the right don't return anything...

IMO the intent is clearer if simply using an `if` in the first place.

I don't know if I'm too picky or if it's something that should be avoided according to the Babylon code writing rules: @sebavan, @RaananW, @deltakosh any thoughts?

Reply:
nope totally agree :-) we should keep it for assignments

Reply:
@Popov72  I agree with you，when the expressions on the right don't return anything，use a `if` statement make the logic clearer